README.md

```diff
@@ -6,7 +6,95 @@ For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/maste
 
 If your project is interested in joining the Node.js Foundation please read the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
 
+## TSC Scope
+
+**The TSC is responsible for Node.js Core project, a number of projects depended upon by Node.js Core, and a number of projects adjacent to Node.js Core.**
+
+The TSC exercises autonomy in managing its responsibilities and seeks agreement from the Node.js Foundation Board on any change to the scope of those responsibilities as defined below.
+
+### Node.js Core
+
+Node.js Core is defined as the code project existing in the repository located at <https://github.com/nodejs/node> which the `node` executable and associated packages are built from, _minus_ the vendored projects included in the codebase, specifically those located in the [deps](https://github.com/nodejs/node/tree/master/deps) and [tools](https://github.com/nodejs/node/tree/master/tools) directories that are copied in from externally maintained repositories.
+
+### Dependent Projects
+
+Additional projects fall within the TSC's scope of responsibility which are dependencies of Node.js Core in that they are **required** to plan, test, build, document and ship Node.js releases.
+
+These projects are located in the following repositories:
+
+* https://github.com/nodejs/build — administration delegated to the Build Working Group
+* https://github.com/nodejs/citgm — administration delegated to the Core Technical Committee, managed by the Smoke Testing Team
+* https://github.com/nodejs/CTC — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/docker-node — administration delegated to the Docker Working Group
+* https://github.com/nodejs/http-parser — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/LTS — administration delegated to the Core Technical Committee, managed by the LTS Team
+* https://github.com/nodejs/node-eps — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/nodejs-dist-indexer — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs-latest-linker — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs-nightly-builder — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs.org — administration delegated to the Website Working Group
+* https://github.com/nodejs/secrets — _private_, administration delegated to the Build Working Group
+* https://github.com/libuv/libuv — currently under Incubation
+
```

Comment:
Per #190 need to add: `https://github.com/nodejs/node-inspect - administration delegated to the Diagnostics Working Group`